compiler/rustc_monomorphize/src/partitioning.rs

```diff
@@ -256,6 +256,7 @@ where
 
         // We can't differentiate something that got inlined.
         let autodiff_active = cfg!(llvm_enzyme)
+            && matches!(mono_item, MonoItem::Fn(_))
```

Comment:
Question: is there a reason this has to be in *this* PR? EDIT: oh this is from #140030.